### PR TITLE
Update to Rails 4.2

### PR DESCRIPTION
**Pull request by adbatista**

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```
Comment (daniloisr) [inline bin/setup]: There is already a setup rake here: https://github.com/Codeminer42/cm42-central/blob/upgrade_rails/lib/tasks/fulcrum.rake#L3

Also, I think that this is would be better placed in another PR (or commit)

Comment (adbatista) [inline bin/setup]: nice, I gonna do this in other PR :)
